Website/src/rendering/src/assets/css/layout/breadcrumb.css

```diff
@@ -1,19 +1,31 @@
 .breadcrumb {
-  @apply max-w-screen-2xl mx-auto overflow-hidden bg-transparent p-7 capitalize;
+  @apply max-w-screen-2xl mx-auto overflow-hidden bg-transparent p-2 capitalize bg-gray-light;
```

Comment:
I remember we ended up discussion that it should look the same (white bg, etc.) but the components below should have less spacing. @scottmulligan ?

![image](https://user-images.githubusercontent.com/1315023/130799521-744adc0f-0cbc-4dd1-b50c-ee3d95710af3.png)

Reply:
Also, padding around breadcrumb has changed?

Reply:
Ah! right,
I thought we were shortening the height of breadcrumb as well.